src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -51,10 +52,14 @@ class VisitService(
   }
 
   fun changeBookedVisit(bookingReference: String, reserveVisitSlotDto: ReserveVisitSlotDto): VisitDto {
+    val visit = visitRepository.findBookedVisit(bookingReference)
```

Comment:
Could create function call validateStart time, and reuse for the 3 cases

Reply:
Created validateVisitStartDate and moved the validation code there.